### PR TITLE
Extend EOS recovery expiration from 1 to 8 hours

### PR DESCRIPTION
**Pull request by bitgopher**
Resolves https://github.com/BitGo/BitGoJS/issues/677 by extending EOS recovery expiration from 1 to 8 hours.

Adds a unit test to demonstrate that the correct expiration date is encoded, by comparing it to the head block time (which is mocked [here](https://github.com/BitGo/BitGoJS/blob/ade4e9ab1459055bf6805e8ca551e98fb44d9a45/modules/core/test/v2/lib/recovery-nocks.ts#L2594)). Also updates the expected packed transaction used across EOS recovery unit tests. (Note: only the first seven bytes are different from the prior packed transaction, which presumably is where the expiration date is encoded.)

### CODE REVIEW

**Comment by bitgopher** [comment]
Note - CI is currently failing on an unrelated jiggler (same error as this PR: https://github.com/BitGo/BitGoJS/pull/675).

**Comment by tylerlevine** [comment]
I have created a ticket for this work - could you also add `Ticket: BG-18622` at the bottom of your commit message? Thanks!

File: modules/core/test/v2/unit/recovery.ts [hunk 1]
```diff
@@ -615,14 +616,16 @@ describe('Recovery:', function() {
 
   describe('Recover EOS', function() {
     let baseCoin;
+    const expectedPackedTrx = '7122315d5e91d408e1b3000000000100a6823403ea3055000000572d3ccdcd0150f3ea2e4cf4bc8300000000a8ed32322150f3ea2e4cf4bc83e0f27c27cc0adf7f40420f000000000004454f53000000000000';
+    const expectedTxId = '99c6a4eedf5cff246314bdc0a053c12d75488df3aa09474bad4ceca88d8b2498';
```
Comment (cooncesean) [inline modules/core/test/v2/unit/recovery.ts]: 👍

**Comment by bitgopher** [comment]
thanks @cooncesean! Just force pushed to sign some commits I had missed.

**Pull request merged**